### DISCUSSION TITLE
cannot filter if set a language manually

### DISCUSSION THREAD

**Discussion by cruvie** [community/community#57994] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

1 search and select a language in the left side bar
<img width="927" alt="iShot_2023-06-14_09 16 27" src="https://github.com/community/community/assets/55651847/cafa03ea-3940-47ee-bbff-1f1c004fbf8b">
<img width="927" alt="iShot_2023-06-14_09 16 39" src="https://github.com/community/community/assets/55651847/4a81ce7c-112e-4fea-aba6-925fe5eb7f77">

2 change language manually in search bar
the result still relate to python rather than go
<img width="882" alt="iShot_2023-06-14_09 13 35" src="https://github.com/community/community/assets/55651847/472c98b4-9852-45d9-9bbf-893854c88109">

**Comment by look** [comment]
Thanks for the report @gkdgo. That's frustrating! I filed a bug with the team.

**Reply by look** [reply]
The bug has been fixed. Thanks again for the report.
